### DISCUSSION TITLE
FreeCodeCamp progress save?

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#75] [Week 3 - Questions]
I started the diner menu on FreeCodeCamp and got to step 58. I then had to leave and when I logged in again, it erased all my progress. I am sick that nothing is saved. Is there a way you can see your history or progress?

**Comment by metasean** [comment]
@kassyoveson 

Were you logged in when you did the exercise?
Are you logged in when trying to review your previous work?
